docs/Migrating.md

```diff
@@ -33,7 +39,13 @@ The versions can be found in
 * If you call `RayPerceptionSensor.PerceiveStatic()` manually, add your inputs to a `RayPerceptionInput`. To get the previous float array output,
  iterate through `RayPerceptionOutput.rayOutputs` and call `RayPerceptionOutput.RayOutput.ToFloatArray()`.
 * Re-import all of your `*.NN` files to work with the updated Barracuda package.
-* Replace all calls to `Agent.GetStepCount()` with `Agent.StepCount`.
+* Replace all calls to `Agent.GetStepCount()` with `Agent.StepCount`
+* Replace the following method with their new equivalent:
```

Comment:
Should we clarify that this is a strong recommendation as these methods will be removed in a later release.